events/src/peer_eventsapi.py

```diff
@@ -79,8 +79,10 @@ def create_custom_config_file_if_not_exists(args):
 
     if not os.path.exists(CUSTOM_CONFIG_FILE):
         with open(CUSTOM_CONFIG_FILE, "w") as f:
-            f.write("{}")
+            with open( CUSTOM_CONFIG_FILE + ".tmp", "w") as f:
```

Comment:
For this to work, the temporary filename must be different each time. Otherwise the same problem that existed before will still be present now in the temporary file.